Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -418,6 +425,11 @@ def PyExec(self):  # noqa
         self._elementSize = self.getProperty("ElementSize").value
         self._num_wl_bins = self.getProperty("NumWavelengthBins").value
 
+        self._interpoTemp = self.getProperty("InterpolateTargetTemp").value
+        self._enableInterpo = False
+        if self._interpoTemp > 0.0:
+            self._enableInterpo = True
+
```

Comment:
``` python
self._enableInterpo = (self._interpoTemp > 0.0)
```
Are we sure interpoTemp cannot be normally non-positive?

Reply:
Temp is in Kelvin, 0 is an appropriate floor